rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -56,4 +56,20 @@
 # define OS_PRIVILEGE_MODE           0
 #endif
 
+#if defined (__ARM_FEATURE_CMSE) &&  (__ARM_FEATURE_CMSE == 3U)
+
+/* Number of process slots (threads may call secure library code) */
+#ifndef MBED_CONF_APP_TZ_PROCESS_STACK_SLOTS
+#define MBED_CONF_APP_TZ_PROCESS_STACK_SLOTS     8U
+#endif
+
+/* Stack size of the secure library code */
+#ifndef MBED_CONF_APP_TZ_PROCESS_STACK_SIZE
```

Comment:
What is this stack for exactly?

Reply:
It is stack for NSC region. i.e Secure library code executed from non-secure process.

Reply:
Might be worth extending the comment ? Are we using code defaults instead of config - (this could be in the config file set, always present if not overriden) ? All in one place, this can however be fixed, as we already have this `mbed_rtx_conf.h` file.

Reply:
@0xc0170 - In case of secure build, we ignore rtos (mbed_rtx_conf.h) hence I am exploring other correct options to update this flag.